### ISSUE TITLE
track 1010EZs with future discharge dates

### ISSUE THREAD

**Issue by batemapf** [department-of-veterans-affairs/va.gov-team#43544]
i would like to understand the volume of 1010EZs that are submitted with future discharge dates. this, likely compared with a rough estimate of the number of transitioning service members for a given period, will help us understand how well the current 1010EZ is serving this user sub-group.

implementation could happen with an analytics event on the FE during submit, or on the BE during validation. or anywhere else that makes sense.

**Comment by hdjustice** [comment]
@batemapf - Can you help with the level of priority for this item?  Are you looking to have this done in the next sprint or 2?

**Comment by markfallows** [comment]
Placing on hold in backlog until we have a better understanding of the priority for this work.

**Comment by hdjustice** [comment]
We can prioritize this when someone has time, or after the Short Form eligibility expansion work.

**Comment by longmd** [comment]
@hdjustice , if we record a GA event, we need to coordinate with the analytics team to actually set up a way for y'all to view it, right?

**Comment by hdjustice** [comment]
@longmd Yes we would need to coordinate with them to ensure the event recorded is being pulled in.  If the event comes in as expected, then we could add it to our [EZ Dashboard](https://analytics.google.com/analytics/web/?authuser=0#/dashboard/gSXa5lOYRyuXpauKyXSYug/a50123418w177519031p184624291/)

**Comment by longmd** [comment]
I have the update ready to put in and test out.... I currently have the event key as `hca-future-discharge-date-submission`. Does that work or do you have a preference for something different? Do we want to coordinate that with them before we merge in the code? I'm not sure on the process for this.

**Comment by hdjustice** [comment]
I posted a note to Michelle Dooley in the analytics channel to see what we would do next.  Will keep you posted!

**Comment by hdjustice** [comment]
Received update from Michelle that this new hca event will be tracked as part of our standard form events.  Once it is active we will see it show up on this [Top Events Report](https://analytics.google.com/analytics/web/#/report/content-event-events/a50123418w177519031p176188361/_u.date00=20220731&_u.date01=20220806&explorer-segmentExplorer.segmentId=analytics.eventLabel&explorer-table.advFilter=%5B%5B0,%22analytics.eventLabel%22,%22PT%22,%22hca--submission%22,0%5D%5D&explorer-table.plotKeys=%5B%5D&explorer-table.secSegmentId=analytics.pagePath&explorer-table.rowCount=25/)

**Comment by batemapf** [comment]
yes please. if you need extra capacity to make that happen and keep everything else on track, please let know and i’ll see if i can pull in someone. thanks!
________________________________
From: Heather Justice ***@***.***>
Sent: Wednesday, July 6, 2022 3:18:44 PM
To: department-of-veterans-affairs/va.gov-team ***@***.***>
Cc: Patrick Bateman ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [department-of-veterans-affairs/va.gov-team] track 1010EZs with future discharge dates (Issue #43544)


@batemapf<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbatemapf&data=05%7C01%7C%7Cb012559f11214a6eced208da5f845c84%7Ce95f1b23abaf45ee821db7ab251ab3bf%7C0%7C0%7C637927319793557397%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2BRpwQ3bB3TsiXSXnP7E89Ea7lRMp79IjlzzuQzAVV4Q%3D&reserved=0> - Can you help with the level of priority for this item? Are you looking to have this done in the next sprint or 2?
